internal/blocksync/pool_test.go

```diff
@@ -110,7 +116,7 @@ func TestBlockPoolBasic(t *testing.T) {
 			if !pool.IsRunning() {
 				return
 			}
-			first, second := pool.PeekTwoBlocks()
+			first, second, _ := pool.PeekTwoBlocks()
```

Comment:
Will we eventually need to handle the ext here too?

Reply:
Not for this particular test, afaict.